springdoc-openapi-webmvc-core/src/main/java/org/springdoc/core/ResponseBuilder.java

```diff
@@ -56,4 +61,12 @@ else if (Callable.class.getName().contentEquals(parameterizedType.getRawType().g
 		return schemaN;
 	}
 
+	private boolean isParameterizedTypeWrapper(Class<?> clazz, ParameterizedType type) {
+		try {
+			Class<?> parameterType = Class.forName(type.getRawType().getTypeName());
+			return clazz.isAssignableFrom(parameterType);
+		} catch (ClassNotFoundException e) {
+			return false;
+		}
+	}
```

Comment:
we want to support any return type inherited from CompletionStage, like ComplatableFuture